src/components/organisms_new/GettingStarted/GettingStarted.tsx

```diff
@@ -0,0 +1,26 @@
+import {useMemo} from 'react';
+
+import {useAppSelector} from '@redux/hooks';
+
+import GettingStartedOverview from '@organismsNew/GettingStarted/GettingStartedOverview';
+
+import {usePaneHeight} from '@hooks/usePaneHeight';
+
+import * as S from './GettingStarted.styled';
+
+const GettingStarted = () => {
+  const integration = useAppSelector(state => state.main.validationIntegration);
+  const height = usePaneHeight();
+
+  const Panel = useMemo(() => {
```

Comment:
I'm not sure this is correct, if you want to memoize the `GettingStartedOverview` component, you should use `React.memo` where you export from it's file:

```ts
export default React.memo(GettingStartedOverview);
```